charts/vllm-kserve/README.md

```diff
@@ -90,6 +90,9 @@ helm upgrade -i [release-name] redhat-ai-services/vllm-kserve \
 For more information on available arguments, see the [vLLM Engine Arguments](https://docs.vllm.ai/en/latest/serving/engine_args.html
 ) documentation.
 
+>[!TIP]
+> Model specific arguments are required to us tooling with vLLM. More info found [here](https://docs.vllm.ai/en/latest/features/tool_calling.html#quickstart)
+
```

Comment:
Can you move this over to this file instead:

https://github.com/redhat-ai-services/helm-charts/blob/main/charts/vllm-kserve/README.md.gotmpl

The README.md file gets generated based on the template file so this would get overwritten.